src/en/administration/first_steps.md

```diff
@@ -30,6 +30,8 @@ To be included in the list of Lemmy instances on [join-lemmy.org](https://join-l
 
 - [x] Federate with at least one instance from the list
 - [x] Have a site description and icon
+- [x] Have open sign-ups
```

Comment:
Or "require registration application"

Reply:
Changed it to "don't have closed sign-ups"